.github/workflows/spelling.yml

```diff
@@ -0,0 +1,24 @@
+name: Spell checking
+on:
+  push:
+    branches:
+      - "**"
+    tags-ignore:
+      - "**"
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '15 * * * *'
```

Comment:
How would this workflow benefit from being scheduled in addition to running on `push`?

Reply:
Push only works if people enable actions in their forks. For security reasons, this isn't a GitHub default.

The normal approach would be to use the pull_request event, but that event doesn't have comment permissions.

There's a newer event, but I can't recall if I've tested it, and the released action doesn't support that.